openadapt/ml/openai/openai_finetune.py

```diff
@@ -0,0 +1,79 @@
+from openadapt.ml.fine_tuning.base_finetuner import FineTuner
+from typing import Optional
+import json
+import subprocess
+
+
+class OpenAIFineTuner(FineTuner):
```

Comment:
Neat pick: May be it is a good idea to put a small `README.md` file with a few lines of example for documentation.

From a new comer point of view,  it is hard to know which method should be called first and which one later.

Reply:
Noted, will add this asap.